snowpack/src/sources/local.ts

```diff
@@ -239,6 +242,17 @@ export default {
     depth = 0,
   ) {
     config = config || _config;
+    // Imports in the same project should never change once resolved. Check the momized cache here to speed up faster repeat page loads.
```

Comment:
Just because I can: "momized" -> "memorized" 😄

Reply:
Or is it memized? Who knows 🤷‍♂️

Reply:
words are hard :)